examples/PagerNode/Sample/PageNode.m

```diff
@@ -21,9 +21,9 @@
 
 @implementation PageNode
 
-- (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
+- (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
```

Comment:
`@note Subclasses that override are committed to manual layout. Therefore, -layout: must be overriden to layout all subnodes or subviews.`

Not needed in this case?